src/main/java/org/gradle/profiler/studio/launcher/StudioLauncher.java

```diff
@@ -64,6 +71,24 @@ private void logLauncherConfiguration(List<String> commandLine) {
         System.out.printf("* Using command line: %s%n%n", String.join(" ", commandLine));
     }
 
+    private Map<String, String> writeIdeaProperties() {
+        if (!studioSandbox.getConfigDir().isPresent()) {
+            System.out.println("* idea.properties can be set only in the sandbox");
+            return Collections.emptyMap();
+        }
+        try {
+            Path ideaPropertiesFile = studioSandbox.getConfigDir().get().resolve("idea.properties").toAbsolutePath();
+            Files.write(ideaPropertiesFile, ideaProperties);
+            return ImmutableMap.<String, String>builder()
+                .put("STUDIO_PROPERTIES", ideaProperties.toString())
+                .put("IDEA_PROPERTIES", ideaProperties.toString())
```

Comment:
:x: I didn't notice before, but we should pass path to a file here, see this change:
```suggestion
                .put("STUDIO_PROPERTIES", ideaPropertiesFile.toString())
                .put("IDEA_PROPERTIES", ideaPropertiesFile.toString())
```